tests/common/config_reload.py

```diff
@@ -44,6 +44,29 @@ def config_system_checks_passed(duthost):
         if int(out['stdout'].strip()) < 120:
             return False
 
+    logging.info("Checking delayed services")
+    # let's cache the delayed services in the function argument
+    if not delayed_services:
+        list_timer_out = duthost.shell(
+            "systemctl list-dependencies --plain sonic-delayed.target | sed '1d'",
+            module_ignore_errors=True
+        )
+        if not list_timer_out["failed"]:
+            check_timer_out = duthost.shell(
+                "systemctl is-enabled %s" % list_timer_out["stdout"].replace("\n", " "),
+                module_ignore_errors=True
+            )
+            if not check_timer_out["failed"]:
+                timers = [_.strip() for _ in list_timer_out["stdout"].strip().splitlines()]
+                states = [_.strip() for _ in check_timer_out["stdout"].strip().splitlines()]
+                delayed_services.extend(
+                    timer.replace("timer", "service") for timer, state in zip(timers, states) if state == "enabled"
+                )
+
+    for service in delayed_services:
+        out = duthost.shell("systemctl is-active %s" % service, module_ignore_errors=True)
+        if out["stdout"].strip().lower() != "active":
+            return False
```

Comment:
This changes the default behavior of `config_system_checks_passed`. The change might cause regression since it's widely used. 
My suggestion is set the default value of `delayed_services` to `None`, which means no need to wait the delay started service.
If an empty list is passed, then generate the list and do the check.

Reply:
@bingwang-ms, updated, now `delayed_services` defaults to empty, and it will be checked only if the DUT is running 202205, which the issue is observed.